use_case_examples/cifar/cifar_brevitas_training/evaluate_one_example_fhe.py

```diff
@@ -77,11 +81,19 @@ def wrapper(*args, **kwargs):
 # cache generated keys through `insecure_key_cache_location`. As the name suggests, these
 # parameters are unsafe and should only be used for debugging in development
 # Multi-parameter strategy is used in order to speed-up the FHE executions
+base_configuration = Configuration()
+
+tlu_optimizer = TLUDeltaBasedOptimizer()
 configuration = Configuration(
     dump_artifacts_on_unexpected_failures=False,
     enable_unsafe_features=True,
     use_insecure_key_cache=True,
     insecure_key_cache_location=KEYGEN_CACHE_DIR,
+    additional_pre_processors=[
```

Comment:
why do we need this? we should implement `compile_torch_model` with `rounding_threshold_bits: { n_bits=AUTO, method = approximate } `